plugin.py

```diff
@@ -26,10 +27,14 @@ def minimum_node_version(cls) -> Tuple[int, int, int]:
     def on_settings_changed(self, settings: DottedDict) -> None:
         super().on_settings_changed(settings)
 
-        if self.get_plugin_setting("dev_environment") == "sublime_text":
+        dev_environment = self.get_plugin_setting("dev_environment")
+
+        if dev_environment in ("sublime_text", "sublime_text_33", "sublime_text_38"):
+            py_ver = "38" if dev_environment == "sublime_text_38" else "33"
```

Comment:
Maybe we should still officially support `sublime_text` which would default to the runtime under which the plugin is running?

Reply:
done in f750f8a